### DISCUSSION TITLE
Welcome to rdb-tools Discussions!

### DISCUSSION THREAD

**Discussion by catcherwong** [catcherwong/rdb-tools#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asd626208641** [comment]
请问rdb-tools能提供参数使得json文件结果除去value打印数据，其它内容保留吗？

**Reply by catcherwong** [reply]
能否描述具体一点？

**Comment by asd626208641** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
